tests/fixtures/joint.py

```diff
@@ -53,17 +54,14 @@ def remove_db_data(self) -> None:
     async def load_test_data(self) -> None:
```

Comment:
In general, we should try to break out of this pattern to populate joint test state in a central fixture. Not something to address here but I put it on the refactoring list. Same I noted for DCS.